### PR TITLE
Fixed broken links and typos

### PR DESCRIPTION
**Pull request by samdutton**
Also noticed that this page links to Tab Flipper and Drink Water samples (and locales.js),  which are not in the [samples directory](https://github.com/GoogleChrome/chrome-extensions-samples).

### CODE REVIEW

File: site/en/docs/extensions/mv3/user_interface/index.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 layout: "layouts/doc-post.njk"
 title: "Design the user interface"
 date: 2018-03-16
-updated: 2021-08-11
+updated: 2021-08-18
 description: UI and design guidelines for Chrome Extensions.
 ---
 
-<-- Note to editors: this is largely identical to the MV2 version, but removes 
+<!-- Note to editors: this is largely identical to the MV2 version, but removes
```
Comment (samthor) [inline site/en/docs/extensions/mv3/user_interface/index.md:9]: whoops

File: site/en/docs/extensions/mv3/user_interface/index.md [hunk 2]
```diff
@@ -267,7 +266,7 @@ the keyword.
 ```
 
 When the user types "nt" into the omnibox, it activates the extension. To signal this to the user,
-it greyscales the provided 16x16 icon and includes it in the omnibox next to the extension name.
+it grayscales the provided 16x16 icon and includes it in the omnibox next to the extension name.
```
Comment (samthor) [inline site/en/docs/extensions/mv3/user_interface/index.md:269]: A controversial change for the ages. (All good, I don't mind if it gets changed)

**Review by samthor — APPROVED** [review]
Samples missing is known, I'll defer to @dotproto.

File: site/en/docs/extensions/mv3/user_interface/index.md (hunk 2 above)
Comment (samdutton) [reply site/en/docs/extensions/mv3/user_interface/index.md:269]: :)

**Comment by samdutton** [comment]
> Samples missing is known, I'll defer to @dotproto.

I wonder if we could add a warning at the top of the document, or link to archived samples (if they exist) with a warning about the version, or just remove the references.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**